packages/lesswrong/components/common/HomeLatestPosts.tsx

```diff
@@ -105,6 +105,7 @@ const HomeLatestPosts = ({classes}: {classes: ClassesType}) => {
   const updateCurrentUser = useUpdateCurrentUser();
   const currentUser = useCurrentUser();
 
+  const userHasRecombeeFrontpage = useRecombeeFrontpage(currentUser);
```

Comment:
Naming is inconsistent with https://github.com/ForumMagnum/ForumMagnum/pull/9034/files#diff-f011fa0e8e4b96adbdd368bacc9ccbbb6d002adbf20b3c8d528ea0439fd62028R24